src/components/link/index.jsx

```diff
@@ -15,7 +15,15 @@ function determineVisuallyHiddenText(supplementalText, opensInNewTab) {
 	return "";
 }
 
-const Link = ({ assistiveHidden, children, className, href, openInNewTab, supplementalText }) => {
+const Link = ({
+	assistiveHidden,
+	children,
+	className,
+	href,
+	openInNewTab,
+	supplementalText,
+	...rest
```

Comment:
did we have trailing comma requirements?

Reply:
😱